clanker-deployer/utils.ts

```diff
@@ -0,0 +1,98 @@
+import { TokenPair } from 'types.js';
+import {
+  ANON_ADDRESS,
+  CB_BTC_ADDRESS,
+  CLANKER_ADDRESS,
+  DEGEN_ADDRESS,
+  HIGHER_ADDRESS,
+  NATIVE_ADDRESS,
+  WETH_ADDRESS,
+} from './constants.js';
+
+export const getDesiredPriceAndPairAddress = (pair: TokenPair) => {
```

Comment:
I don't know if we want to provide these hardcoded variables, I think one example one is enough. I can see drift happening in the actual prices causing issues for people in the future. I'm ok if you want to leave it in tho